google/resource_compute_project_metadata.go

```diff
@@ -87,6 +90,18 @@ func resourceComputeProjectMetadataCreate(d *schema.ResourceData, meta interface
 	return resourceComputeProjectMetadataRead(d, meta)
 }
 
+// This function differs from flattenMetadata only in that it takes
+// compute.metadata rather than computeBeta.metadata as an argument. It should
+// be removed in favour of flattenMetadata if/when this resource gets beta
+// support.
+func flattenCommonInstanceMetadata(metadata *compute.Metadata) map[string]string {
```

Comment:
There are a whole bunch of metadata expanders/flatteners floating around (here, resource_compute_instance, utils.go, metadata.go)- is it possible to have just one of each that live in the same file? (my vote is for metadata.go)

Reply:
SGTM, will change.